www/index.html

```diff
@@ -95,7 +95,8 @@
     <script src="js/common/place-detail.js"></script>
     <script src="js/common/services.js"></script>
     <script src="js/survey/launch.js"></script>
-    <script src="js/survey/enketo_launch.js"></script>
+    <script src="js/survey/enketo-survey.js"></script>
```

Comment:
I am not sure why you needed to rename `enketo_launch.js` to `enketo-survey.js` but it was a small file so I am not going to hold up the change for this.